projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
I was initially against this placeholder, but it's growing on me. The flexibility it provides will be important no doubt. Not sure if we should go with `cluster` though. `env` perhaps?